backend/tests/unit/dal/coverlet.runsettings

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<RunSettings>
+  <DataCollectionRunSettings>
+    <DataCollectors>
+      <DataCollector friendlyName="XPlat code coverage">
+        <Configuration>
+          <Format>json,opencover</Format>
+          <MergeWith>../TestResults/coverage.json</MergeWith>
```

Comment:
The merge with the `API` report happens due to this config setting.

**NOTE** - if more test sub-projects are added later, we will need to keep merging with the other ones and execute test serially. Executing tests in parallel will not work with code coverage unfortunately - a restriction of the tool we are using.

Reply:
This is good to know.